### PR TITLE
#1686 - Add togrep methods for AbstractZonotopes

### LINKED ISSUE #1686: Add togrep methods for AbstractZonotopes
> Should we have a togrep method for AbstractZonotopes which returns a Zonotope/GParallelotope/Parallelotope depending on the type? (Actually, all subtypes except for Zonotope are parallelotopes.)

Ok, these conversions can be added as well.

_Originally posted by @mforets in https://github.com/JuliaReach/LazySets.jl/pull/1632#issuecomment-530735813_

### PR DESCRIPTION
**Pull request by schillic**
Closes #1686.

### CODE REVIEW

File: src/Interfaces/AbstractZonotope.jl [hunk 1]
```diff
@@ -118,6 +119,25 @@ function generators_fallback(Z::AbstractZonotope{N}) where {N<:Real}
...
+
+### Input
+
+- `Z` -- zonotopic set
+
+### Output
+
+The same set in generator representation.
+This fallback implementation returns a `Zonotope`; in general, other generator
+representations may be returned.
```
Comment (mforets) [inline src/Interfaces/AbstractZonotope.jl]: > ; in general, other generator representations may be returned.

I don't find the part after the semicolon very clear.. can we delete it? Or say that to use other representation, try using `convert` method directly.

Comment (schillic) [reply src/Interfaces/AbstractZonotope.jl]: I wanted to say here: do not rely on the fact that `togrep` *always* returns a `Zonotope`. This is not about `convert`ing.

Comment (mforets) [reply src/Interfaces/AbstractZonotope.jl]: Ah, i see what you mean. The source of my confusion was the part ".. may be returned." (..by whom? Answer: *not* by this function; by other, specific implementations of `togrep` ).

Comment (schillic) [reply src/Interfaces/AbstractZonotope.jl]: Okay, would the following help?
```
This fallback implementation returns a `Zonotope`; however, more specific
implementations may return other generator representations.
```

Comment (mforets) [reply src/Interfaces/AbstractZonotope.jl]: yes

**Pull request merged**